### ISSUE TITLE
Design and print stickers

### ISSUE THREAD

**Issue by samcunliffe** [UCL-ARC/python-tooling#469]
### What needs to be done?

Adapt the logo into a hex? or shaped sticker.
Add a nice short URL
Print.

### Blocked by

- #468

**Comment by matt-graham** [comment]
As an alternative to a short URL we could also tag with `gh:ucl-arc/python-tooling` which probably would give most people enough info to construct URL and also tells people what they need to use `cookiecutter` command. I guess till not that short though!

**Comment by samcunliffe** [comment]
I'll give them one more prod, after that I would propose we abandon the idea of a short URL.

**Comment by samcunliffe** [comment]
Just to dump some conversation here:

* Chris gave the OK to use ARC credit card for ballpark £50 (around 100 units) from [Dignate](https://diginate.com/ideas/create-laptop-hex-sticker/) (UK-based company that are slightly cheaper than the other place I got a quote from).
* The inclination, IIRC was to go with [hex-sticker compliant](https://github.com/terinjokes/StickersStandard).

**Comment by samcunliffe** [comment]
![Image](https://github.com/user-attachments/assets/e097c80b-19e0-4852-9852-cf043bd71b68) ![Image](https://github.com/user-attachments/assets/c2f227e8-5c36-4bea-9ba5-d9b54899b514) ![Image](https://github.com/user-attachments/assets/5db70c63-d0c4-407a-8700-317fc7be2bb0)

(Let's call them version 0: a, b, and c ... can I get some opinions plz?)

**Comment by Saransh-cpp** [comment]
I like having the link/reference

**Comment by matt-graham** [comment]
Agree with Saransh that having the template link / reference is good. Just to note we probably would want to convert the text to a path in the SVG in Inkscape before ordering for printing as the font used may not be available and another font will be used in place. I don't seem to be able to open the embedded SVGs in https://github.com/UCL-ARC/python-tooling/issues/469#issuecomment-2631851186 in Inkscape, but in Firefox on Windows the rightmost is showing the text in a serif font while my memory of viewing it on an Ubuntu machine was it was in a san-serif font?

**Comment by samcunliffe** [comment]
> I  don't seem to be able to open the embedded SVGs

Yeah. Something's gone wrong with them indeed. The font is not what I intended. I'll do another round of voting and be sure to embed/raster the fonts. Also: we need some bleed margins.

**Comment by samcunliffe** [comment]
![v1a](https://github.com/user-attachments/assets/ae2045e3-1bd6-483d-985f-207c1cdad83e) ![v1b](https://github.com/user-attachments/assets/4c76c8b5-fecf-47b4-830b-79220bb46852) ![v1c](https://github.com/user-attachments/assets/de8780b6-b958-425e-9693-5f5bdb31f44c)

v1: a, b, and c

(c is very dark blue)
(@matt-graham can you try downloading one? did the fonts preserve correctly?)

**Comment by matt-graham** [comment]
I can now open these in Inkscape (or at least I tried the final one, assuming the rest will also!). In terms of fonts - this is how I see the text

![Image](https://github.com/user-attachments/assets/e831dee5-c8bd-43e7-8eb3-936fc2f2562b)

I think it's still text rather than a path (and not sure you can embed font in SVG?) but as I have Arial font (which appears to the the `font-family` specification in file) it is rendering correctly for me. If you to run `Path > Object to path` in Inkscape menu it would force it to become a path which would be ensure it renders the same irrespective of the fonts installed on the system (but loose the ability to edit the text).

**Comment by samcunliffe** [comment]
Thanks! Yeah I swapped to Ariel because I thought it looked better, in the end. Indeed it was still text. Have now converted to path.

**Comment by samcunliffe** [comment]
Ordered.

**Comment by samcunliffe** [comment]
Printed. Done.

----

Final SVG proof (w/ bleed):
![Image](https://github.com/user-attachments/assets/90b163d1-1ada-4978-bd79-e1ce327f8d88)

Diginate wanted a PDF:
[v1b.pdf](https://github.com/user-attachments/files/18842696/v1b.pdf)
